### PR TITLE
✨ Added port to project.pros:

### PR DESCRIPTION
**Pull request by Aniket-Rai1**
#### Summary:
This pr adds the port option to project.pros

#### Motivation:
https://github.com/purduesigbots/pros-cli/issues/223

### CODE REVIEW

File: pros/cli/conductor.py [hunk 1]
```diff
@@ -12,7 +12,7 @@ def conductor_cli():
     pass
 
 
-@conductor_cli.group(cls=PROSGroup, aliases=['cond', 'c', 'conduct'], short_help='Perform project management for PROS')
+@conductor_cli.group(cls=PROSGroup, aliases=['cond', 'c', 'conduct'], short_help='aniket')
```
Comment (BennyBot) [inline pros/cli/conductor.py]:
```suggestion
@conductor_cli.group(cls=PROSGroup, aliases=['cond', 'c', 'conduct'], short_help='Perform project management for PROS')
```

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -47,7 +47,7 @@ jobs:
       if: matrix.os == 'macos-latest'
         
     - name: Install Requirements
-      run: python3 -m pip install --upgrade pip && pip3 install wheel && pip3 install -r requirements.txt && pip3 uninstall -y typing
+      run: python3 -m pip install --upgrade pip && pip3 install wheel && pip3 install -r requirements.txt && pip3 uninstall -y typing && pip3 uninstall -y charset_normalizer
```
Comment (BennyBot) [inline .github/workflows/main.yml]:
```suggestion
      run: python3 -m pip install --upgrade pip && pip3 install wheel && pip3 install -r requirements.txt && pip3 uninstall -y typing
```

File: pros/cli/upload.py [hunk 3]
```diff
@@ -70,6 +70,8 @@ def upload(path: Optional[str], project: Optional[c.Project], port: str, **kwarg
 
         # apply upload_options as a template
         options = dict(**project.upload_options)
+        if 'port' in options and kwargs.get('port', None) is None:
+            kwargs.pop('port')
```
Comment (Aniket-Rai1) [inline pros/cli/upload.py]:
```suggestion
        if 'port' in options and port is None:
            port = options.get('port', None)
```

**Review by BennyBot — APPROVED** [review]
LGTM

**Review by ayushuk — CHANGES_REQUESTED** [review]
I prefer we test this and provide screenshots if possible. I know the change is small but I don't want any small edge cases getting through

**Review by ayushuk — APPROVED** [review]
Tested on mac. LGTM
